### DISCUSSION TITLE
Change background of semaphore gui

### DISCUSSION THREAD

**Discussion by get2aakash** [semaphoreui/semaphore#2089] [Q&A]
Is it possible to change the background picture of semaphore login screen ..I have requirement to put my organisation logo..
Thanks

**Comment by tboerger** [comment]
Currently there is nothing related to customization, but maybe you want to propose something?

**Reply by get2aakash** [reply]
It would be good to be able to place background image and organisation logo..Both can be mounted as docker volume to the path and replace the default one.
